MontyHallTests/PlayerChoiceValidatorTests.cs

```diff
@@ -0,0 +1,74 @@
+using MontyHallv2.DoorCreation;
+using MontyHallv2.GameModes;
+using MontyHallv2.Validation;
+
+namespace TestProject1.moq;
+
+public class PlayerChoiceValidatorTests
+{
+    private readonly ConsoleGame _consoleGame = new();
+
+    [Fact]
+    public void GivenGetDoorSelectionFromUserIsCalled_WhenUserDoorChoiceIsInvalid_ThenValidatorWillReturnFalse()
+    {
+        //Arrange
+        var invalidDoorChoice = "4\n2";
+        var stringReader = new StringReader(invalidDoorChoice);
+        Console.SetIn(stringReader);
+
+        //Act
+        var userDoorSelection = _consoleGame.GetDoorSelectionFromUser();
+        var isUserDoorSelectionValid = PlayerChoiceValidator.ValidateUserDoorSelection(userDoorSelection);
+
+        //Assert
+        Assert.False(isUserDoorSelectionValid);
```

Comment:
does the validator need to know about the way that the player choice is input? what if we just test the validator in isolation e.g.:
```suggestion
    public void GivenDoorChoice_WhenUserDoorChoiceIsInvalid_ThenValidatorWillReturnFalse()
    {
        //Arrange
        var userDoorSelection = "4";

        //Act
        var isUserDoorSelectionValid = PlayerChoiceValidator.ValidateUserDoorSelection(userDoorSelection);

        //Assert
        Assert.False(isUserDoorSelectionValid);
```